.git-blame-ignore-revs

```diff
@@ -0,0 +1,4 @@
+# To apply this file locally, run: `git config blame.ignoreRevsFile .git-blame-ignore-revs`
+
+# applied treefmt for the first time
+3391af283a72871b3afe23dd721e35c52daba241
```

Comment:
Not sure if it's all that helpful and probably not worth keeping it around. CC @rhyslbw

Reply:
This is common recommended practice and leverages the corresponding [GitHub feature](https://github.blog/changelog/2022-03-24-ignore-commits-in-the-blame-view-beta/).

Reply:
Oh that's a cool feature! Looks good here